ui/src/views/plugins/ApiDocsPlugin.vue

```diff
@@ -0,0 +1,210 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <div>
+    <resource-layout>
+      <template #left>
+        <a-card :bordered="false">
+          <a-auto-complete
+            v-model:value="query"
+            :options="options"
+            style="width: 100%"
+            >
+            <a-input-search
+                size="default"
+                :placeholder="$t('label.search')"
+                v-model:value="query"
+                allow-clear
+                enter-button
+                >
+                <template #prefix><search-outlined /></template>
+            </a-input-search>
+          </a-auto-complete>
+          <a-list style="margin-top: 12px; height:580px; overflow-y: scroll;" size="small" :data-source="Object.keys($store.getters.apis).sort()">
+            <template #renderItem="{ item }">
+              <a>
+              <a-list-item v-if="item.toLowerCase().includes(query.toLowerCase())" @click="showApi(item)">
+                <span v-if="selectedApi === item">
+                  <strong>{{ item }}</strong> <a-tag v-if="$store.getters.apis[item].isasync" color="blue">async</a-tag>
+                </span>
+                <span v-else>
+                  {{ item }} <a-tag v-if="$store.getters.apis[item].isasync" color="blue">async</a-tag>
+                </span>
+              </a-list-item>
+              </a>
+            </template>
+          </a-list>
+          <a-divider style="margin-bottom: 12px" />
+          <span>{{ Object.keys($store.getters.apis).length }} {{ $t('label.api.docs.count') }}</span>
+        </a-card>
+      </template>
+      <template #right>
+        <a-card
+          class="spin-content"
+          :bordered="true"
+          style="width: 100%; overflow-x: auto">
+          <span v-if="selectedApi && selectedApi in $store.getters.apis">
+            <h2>{{ selectedApi }} <a-tag v-if="$store.getters.apis[selectedApi].isasync" color="blue">Asynchronous API</a-tag></h2>
+            <p>{{ $store.getters.apis[selectedApi].description }}</p>
+            <h3>Request parameters:</h3>
+            <a-table
+              :columns="[{title: 'Parameter Name', dataIndex: 'name'}, {title: 'Required', dataIndex: 'required'}, {title: 'Type', dataIndex: 'type'}, {title: 'Description', dataIndex: 'description'}]"
+              :data-source="selectedParams"
+              :pagination="false"
+              size="small">
+              <template #bodyCell="{text, record}">
+                <span v-if="record.required === true"> <strong>{{ text }}</strong> </span>
+                <span v-else>  {{ text }} </span>
+              </template>
+            </a-table>
+            <br/>
+            <h3>Response keys:</h3>
```

Comment:
```suggestion
            <h3>Response parameters:</h3>
```

can use label here, instead of hardcoded text